### PR TITLE
refactor a cleanUp function

### PR DESCRIPTION
**Pull request by usernameisnull**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: pkg/kwokctl/cmd/create/cluster/cluster.go [hunk 1]
```diff
@@ -127,7 +127,6 @@ func runE(ctx context.Context, flags *flagpole) error {
 		ctx, cancel = context.WithTimeout(ctx, flags.Timeout)
 		defer cancel()
 	}
-
```
Comment (wzshiming) [inline pkg/kwokctl/cmd/create/cluster/cluster.go:130]: Keep the empty line

Comment (usernameisnull) [reply pkg/kwokctl/cmd/create/cluster/cluster.go]: done，please review again

**Comment by wzshiming** [comment]
May you `git commit --amend` commit and push again?

**Pull request merged**